### PR TITLE
Adds sort on tasks page

### LINKED ISSUE #49: Re-enable sorting on tasks page

### PR DESCRIPTION
**Pull request by hannako**
Closes #49

### CODE REVIEW

File: app/helpers/tasks_helper.rb [hunk 1]
```diff
@@ -1,2 +1,63 @@
 module TasksHelper
+
```
Comment (govuklaurence) [inline app/helpers/tasks_helper.rb]: This could be refactored, but for now it's only complaining about duplication... we'll come back to it later

Comment (hannako) [inline app/helpers/tasks_helper.rb]: am mid refactor, one sec

Comment (hannako) [inline app/helpers/tasks_helper.rb]: This is still pretty terrible, sorry.

**Comment by govuklaurence** [comment]
Will pair with you tomorrow, we'll see what we can do

**Pull request merged**